src/main/java/meteordevelopment/meteorclient/systems/modules/combat/SelfTrap.java

```diff
@@ -182,29 +182,29 @@ private void findPlacePos(Block block) {
         BlockPos pos = mc.player.getBlockPos();
 
         switch (topPlacement.get()) {
```

Comment:
Why are we switching to enhanced switch casing, this is also not necessary for the 1.20 port